alternator/executor.cc

```diff
@@ -1679,6 +1679,10 @@ rmw_operation::parse_returnvalues_on_condition_check_failure(const rjson::value&
         throw api_error::validation(fmt::format("Unrecognized value for ReturnValuesOnConditionCheckFailure: {}", s));
     }
 }
+static replica::table_stats& get_table_stats_from_schema(service::storage_proxy& sp, const schema& schema) {
+    replica::table& table = sp.local_db().find_column_family(schema.id());
+    return table.get_stats();
+}
```

Comment:
table is a replica-side object, but alternator lives in the coordinator.

We should either make the measurements fully coordinator-side or fully replica-side, not mixed.

Reply:
You are right, but Alternator does not have its own concept of tables. It uses the replica tables for information (and bookkeeping, for that matter)

Reply:
There is data_dictionary.

Reply:
I discussed it with @nyh. All Alternator metrics are at the coordinator level, but there is no mechanism in Alternator that follows the tables' lifecycle. When the Alternator needs table information, it goes to a local table, in practice to the replica.